library/src/main/java/nl/dionsegijn/steppertouch/StepperTouch.kt

```diff
@@ -52,6 +52,7 @@ class StepperTouch : FrameLayout, OnStepCallback {
     private var stepperButtonColor = R.color.stepper_button
     private var stepperTextSize = 20
     private var allowNegativeStepper = true
+    private var allowPositiveStepper = true
```

Comment:
Let me come back at the naming again, I gave it some thought. 

I'd rather still like to use the `hide` prefix instead of allow since it's a little more descriptive of what it will visually do.

`hideMinus` & `hidePlus`. It's also in negative form, so instead of `show` or `allow` it's emphasising that it's not the default behavior. 
Also by calling it Minus and Plus it feels like we don't need to call it stepper anymore.

Let me know what you think

Reply:
In my opinion show and hide is not descriptive, because we use it in everywhere to just show and hide a visual element, but here we additionally disable them,
By the way, I think we should conclude this conversation ASAP,
about Plus and Minus, I disagree with you because you used negative and positive in other variables, and we shouldn't choose name base on their characters count, we should keep the names consistent,
and about stepper you're right.
what do you think?

Reply:
> about Plus and Minus, I disagree with you because you used negative and positive in other variables, and we shouldn't choose name base on their characters count, we should keep the names consistent

Yes, good point! 

Ok, let's rename it to allowPositive and allowNegative then

Reply:
Done!